app/screens/home-screen/home-screen.tsx

```diff
@@ -609,16 +655,41 @@ const useStyles = makeStyles(({ colors }) => ({
     maxWidth: "25%",
     flexGrow: 1,
   },
+  balanceContainer: {
+    marginBottom: 30,
+    marginTop: 7,
```

Comment:
did you test this with different screen sizes and in iOS? (applies for all fixed values in margins or paddings)